### ISSUE TITLE
Unable to Download Pixiv Images

### ISSUE THREAD

**Issue by Shagiakhmetov** [Nandaka/PixivUtil2#688]
Good afternoon.

I downloaded the latest version of PixivDownloader (202000428-beta1) and ran into the following problem:
FanBox downloads fine, but any downloads from Pixiv are not possible. An example is below.

```
PixivDownloader2 version 202000428-beta1
https://github.com/Nandaka/PixivUtil2/releases
Donate at https://bit.ly/PixivUtilDonation
Reading E:\PixivUtil\config.ini ...
done.
Using default DB Path: E:\PixivUtil\db.sqlite
Creating database... done.
Only process members where the last update is >= 7 days ago
Using Username: shagiahmetov
Trying to log in with saved cookie
Login successful.
My User Id: NNNNNNNN.
Premium User: True.
Trying to log in FANBOX with saved cookie
Login successful.
PixivDownloader2 version 202000428-beta1
https://github.com/Nandaka/PixivUtil2/releases
Donate at https://bit.ly/PixivUtilDonation
1. Download by member_id
2. Download by image_id
3. Download by tags
4. Download from list
5. Download from bookmarked artists (/bookmark.php?type=user)
6. Download from bookmarked images (/bookmark.php)
7. Download from tags list
8. Download new illust from bookmarked members (/bookmark_new_illust.php)
9. Download by Title/Caption
10. Download by Tag and Member Id
11. Download Member Bookmark (/bookmark.php?id=)
12. Download by Group Id
------------------------
f1. Download from supported artists (FANBOX)
f2. Download by artist/creator id (FANBOX)
f3. Download by post id (FANBOX)
f4. Download from followed artists (FANBOX)
------------------------
d. Manage database
e. Export online bookmark
m. Export online user bookmark
i. Import list file
r. Reload config.ini
p. Print config.ini
x. Exit
Input: 2
Image ids: 81021029
Processing Image Id: 81021029
Image ID (81021029): Expecting value: line 1 column 1 (char 0)
Stack Trace: (<class 'json.decoder.JSONDecodeError'>, JSONDecodeError('Expecting value: line 1 column 1 (char 0)'), <traceback object at 0x14D047A8>)
Traceback (most recent call last):
  File "PixivUtil2.py", line 662, in process_image
  File "PixivBrowserFactory.pyc", line 517, in getImagePage
  File "PixivBrowserFactory.pyc", line 546, in getMemberInfoWhitecube
  File "json\__init__.pyc", line 357, in loads
  File "json\decoder.pyc", line 337, in decode
  File "json\decoder.pyc", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by Shagiakhmetov** [comment]
UPD.

My download from FanBox works fine both in the "f1" mode and in the "f2" mode, and in the "f4" mode.

The problem is only with the "f3" mode:

```
Input: f3
Post ids = 986608
Traceback (most recent call last):
  File "PixivUtil2.py", line 2524, in main
  File "PixivUtil2.py", line 2256, in main_loop
  File "PixivUtil2.py", line 1964, in menu_fanbox_download_by_post_id
  File "PixivBrowserFactory.pyc", line 881, in fanboxGetPost
AttributeError: 'NoneType' object has no attribute 'creatorId'
press enter to exit.
```

**Comment by Nandaka** [comment]
Should be fixed in #683 
Try the latest releases?

**Comment by Shagiakhmetov** [comment]
In beta-2 bugs fixed.
Thank.
